docker-compose-installed.yaml

```diff
@@ -44,10 +52,7 @@ services:
     entrypoint: sawtooth-rest-api --connect tcp://validator:9704 --bind rest-api:9708 -vv
 
   shell:
-    build:
-      context: .
-      dockerfile: cli/Dockerfile-installed
-    image: sawtooth-sabre-cli:${ISOLATION_ID}
+    image: hyperledger/sawtooth-all:1.0
```

Comment:
This is breaking your build. was the change intentionally?

Reply:
Thanks for pointing the issue. I changed changed the Jenkinsfile too to reflect that `sabre-cli` entry -- as opposed to `shell` -- should be responsible for building the sabre-cli container now. will see how the jenkins test does with this change

Reply:
@agunde406 to answer your first question, the change was intentional. The Jenkins test is still failing because the Jenkinsfiles that is picked up to test the PR is the one from the master branch. I'm not sure what to do from there 🤔